go.mod

```diff
@@ -62,6 +62,7 @@ replace (
 	// controller runtime
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200331152225-585af27e34fd // release-4.5
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
+	github.com/operator-framework/api => github.com/awgreene/api v0.0.4
```

Comment:
TODO: If we are not able to unblock https://github.com/operator-framework/operator-lifecycle-manager/pull/2368 then revert the CR bump in https://github.com/operator-framework/api/pull/154